rise-financial-es6.js

```diff
@@ -83,8 +108,63 @@
       const instruments = snapshot.val();
 
       this._instruments = instruments ? instruments : {};
+      this._instrumentsReceived = true;
+      this.go();
+    }
+
+    /***************************************** REAL-TIME ******************************************/
+
+    _getRealTimeUrl() {
+      return "http://contentfinancial2.appspot.com/data";
+    }
+
+    _getRealTimeParams( instruments, fields = [] ) {
+      return Object.assign( {},
+        {
+          id: this.displayId,
+          code: this._getSymbols( instruments ),
+          tqx: "out:json;responseHandler:callback",
+        },
+        fields.length > 0 ? { tq: this._getRealTimeQueryString( fields ) } : null );
+    }
+
+    _getRealTimeQueryString( fields = [] ) {
+      if ( fields.length === 0 ) {
+        return "";
+      }
+
+      return `select ${ fields.join( "," ) }`;
+    }
 
-      console.log( this._instruments );  // eslint-disable-line no-console
+    _getRealTimeData( instruments, fields = [] ) {
+      const realTime = this.$.realTime;
+
+      realTime.url = this._getRealTimeUrl();
+      realTime.params = this._getRealTimeParams( instruments, fields );
```

Comment:
@settinghead That's correct. It's what the [`auto` attribute](https://github.com/Rise-Vision/rise-financial/pull/6/files#diff-1c4ad2054a29419e2f296d322fd3b64eR8) does.